recipes/spirv-headers/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SpirvheadersConan(ConanFile):
+    name = "spirv-headers"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Headers"
+    description = "SPIRV-Headers"
+    topics = ("conan", "sprv", "vulkan")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+
+    license = "MIT-KhronosGroup"
+
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "SPIRV-Headers-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
```

Comment:
You can cache cmake to avoid reconfigure during package step, have a look in https://github.com/conan-io/conan-center-index/blob/master/recipes/bullet3/all/conanfile.py for example.

Reply:
Should I though? Will it make a huge difference? I copied the recipe from another header-only lib.

Reply:
Pretty sure that header only libraries using CMake spend most of their build/install time (which is fast anyway) in CMake configure.

Reply:
You're right. I just did a test and it was a bit faster. I'll make the change.